docs/platform/deploying-airbyte/troubleshoot-deploy.md

```diff
@@ -164,7 +164,11 @@ Upgrade to version `>0.15.1`
 
 Version `0.15.0` had a bug when users have `secrets.yaml` file. You must upgrade your `abctl` to fix this issue.
 
----
+### Make sure you have disabled secure cookies
+
+- Error: `Your credentials were correct, but the server failed to set a cookie. You appear to have deployed over HTTP. Make sure you have disabled secure cookies.`
+- Cause: You have deployed Airbyte to an insecure (non-HTTPS) host, but you haven't turned off secure cookies.
+- Fix: Deploy Airbyte again and set the `--insecure-cookies` flag. For example, `abctl local install --host example.com --insecure-cookies`.
```

Comment:
FYI there was a recent fix for --insecure-cookies. Latest version should now be working. See - https://github.com/airbytehq/airbyte/issues/61586